MEME/MEME/Model/ModelHomeView/ViewModel/ModelHomeViewModel.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  ModelHomeViewModel.swift
+//  MEME
+//
+//  Created by 정민지 on 4/3/24.
+//
+
+import RxSwift
+import Foundation
+
+class ModelHomeViewModel: ViewModel {
+    private let disposeBag = DisposeBag()
+    private let reservationManager = ReservationManager.shared
+    
+    //MARK: - Input
+    struct Input {
+            let modelReservationsTrigger: Observable<Int>
+            let getRecommendArtistByReviewTrigger: Observable<Void>
+            let getRecommendArtistByRecentTrigger: Observable<Void>
+        }
+        
+        struct Output {
+            let modelReservations: Observable<[ReservationData]>
+            let recommendArtistByReview: Observable<[Portfolio]>
+            let recommendArtistByRecent: Observable<[Portfolio]>
+        }
+    
+    //MARK: - transform
+    func transform(_ input: Input) -> Output {
+        let modelReservations = input.modelReservationsTrigger
+            .flatMapLatest { modelId -> Observable<[ReservationData]> in
+                return self.fetchModelReservations(modelId: modelId)
+                    .catchAndReturn([])
+            }
+        
+        let recommendArtistByReview = input.getRecommendArtistByReviewTrigger
+            .flatMapLatest { _ in
+                return self.fetchRecommendArtistByReview()
+                    .catchAndReturn([])
+            }
+        
+        let recommendArtistByRecent = input.getRecommendArtistByRecentTrigger
+            .flatMapLatest { _ in
+                return self.fetchRecommendArtistByRecent()
+                    .catchAndReturn([])
+            }
+        
+        return Output(
+            modelReservations: modelReservations,
+            recommendArtistByReview: recommendArtistByReview,
+            recommendArtistByRecent: recommendArtistByRecent
+        )
+    }
+    
+    //MARK: - API
+    private func fetchModelReservations(modelId: Int) -> Observable<[ReservationData]> {
+        return Observable.create { observer in
+            self.reservationManager.getModelReservation(modelId: modelId) { result in
+                switch result {
+                case .success(let reservationDTO):
+                    let filteredData = reservationDTO.data?.filter { reservationData in
+                        if let date = self.dateFromString(reservationData.reservationDate),
+                           self.isToday(date),
+                           reservationData.status == "EXPECTED" {
```

Comment:
Core/Model/ReservationState에 가지면 
```
import Foundation

enum ReservationState: String {
    case EXPECTED
    case COMPLETE
    case CANCEL
}
```
와 같은 열거형이 있습니다!
```suggestion
                           reservationData.status == ReservationState.EXPECTED.rawValue {
```
와 같이 사용하면 어떨까요?